### ISSUE TITLE
More details

### ISSUE THREAD

**Issue by Zhiwu6** [miraclefactory/kernel-transformer#1]
I would like to know more details about this kernel-Transformer. Has there been a paper published on this?

**Comment by yuelinxin** [comment]
No, this is recurrently just a rough idea, it works very similarly as Swin Transformer, but with some added details in representation and flexibility of architecture.
